src/parser/template/parameters.ts

```diff
@@ -1,155 +1,33 @@
-import { parseNumber } from '../private/types';
-import { schema } from '../schema';
+import { CfnParameterProps } from 'aws-cdk-lib';
+import {
+  assertBoolean,
+  assertList,
+  assertNumber,
+  assertObject,
+  assertString,
+} from '../private/types';
+import { ifField } from './expression';
 
-export type ContextValue = string | string[] | symbol;
+export interface TemplateParameter extends CfnParameterProps {}
 
-export class TemplateParameters {
```

Comment:
All of the remove code was unused and appears to be intended for local resolution of parameter values. We don't need that here.